lib/bank_account.rb

```diff
@@ -0,0 +1,29 @@
+class BankAccount
+ 
+  attr_accessor :name, :job, :email, :address, :balance 
```

Comment:
`:name, :job, :email, :address` sound more like fields for an User than a bank account, you should create a new model for these attributes